librz/arch/isa/pic/pic18_analysis.inc

```diff
@@ -0,0 +1,377 @@
+// SPDX-FileCopyrightText: 2015-2018 oddcoder <ahmedsoliman@oddcoder.com>
+// SPDX-FileCopyrightText: 2015-2018 thestr4ng3r <info@florianmaerkl.de>
+// SPDX-FileCopyrightText: 2015-2018 courk <courk@courk.cc>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "pic_pic18.h"
+#include "pic18_esil.inc"
+
+typedef struct {
+	enum {
+		ILOpEff_None,
+		ILOpEff_PostDec,
+		ILOpEff_PostInc,
+		ILOpEff_PreInc,
+	} tag;
+	char fsr[8];
+} ILOpEff;
+
+typedef struct {
+	const Pic18Op *op;
+	const HtSU *mm;
+	ILOpEff eff;
+} Pic18ILContext;
+
+#include "pic18_il.inc"
+
+static void pic18_cond_branch(RzAnalysisOp *aop, ut64 addr, const ut8 *buf) {
+	aop->type = RZ_ANALYSIS_OP_TYPE_CJMP;
+	aop->jump = addr + 2 + 2 * (*(ut16 *)buf & 0xff);
```

Comment:
Shouldn't be endian-independent function used here instead of dereferencing pointer?